README.md

```diff
@@ -15,175 +15,10 @@ With [Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/app
-    )
-```
-
-If you want to execute this same code in your local environment machine, just replace the `MSIAuthentication` class with `ServicePrincipalCredentials` or any credentials class available for Python. 
-An overview of credentials features is available here: [https://docs.microsoft.com/python/azure/python-sdk-azure-authenticate](https://docs.microsoft.com/python/azure/python-sdk-azure-authenticate)
-
-If you need a fallback mechanism to allow this code to switch automatically from MSI to another approach, you can test for environment variables:
-
-```python
-if "APPSETTING_WEBSITE_SITE_NAME" in os.environ:
-    return MSIAuthentication(
-        resource='https://vault.azure.net'
-    )
-else:
-    return ServicePrincipalCredentials(
-        client_id=os.environ['AZURE_CLIENT_ID'],
-        secret=os.environ['AZURE_CLIENT_SECRET'],
-        tenant=os.environ['AZURE_TENANT_ID'],
-        resource='https://vault.azure.net'
-    )
-```
-
-## Summary
-
-The web app was successfully able to get a secret at runtime from Azure Key Vault using your developer account during development, and using MSI when deployed to Azure, without any code change between local development environment and Azure. 
-As a result, you did not have to explicitly handle a service principal credential to authenticate to Azure AD to get a token to call Key Vault. You do not have to worry about renewing the service principal credential either, since MSI takes care of that.
-
-## Azure Functions
-
-Azure Functions being powered by Azure WebApp, MSI is also available. Just copy the content of `run_example` into your Azure Functions with the right import.
-
-## Troubleshooting
-
-### Common issues when deployed to Azure App Service:
-
-1. I see "The page cannot be displayed because an internal server error has occurred.", even on the "ping" endpoint
-
-Make sure you have installed a Python extension for WebApp (see Step 1). If not, this tutorial explains [how to update Python using an extension on Azure WebApp](https://docs.microsoft.com/visualstudio/python/managing-python-on-azure-app-service).
- The sample here works directly if you install the extension "Python 3.6.2 x86". Edit the `web.config` file if you wish to use another version of Python.
-
-1. MSI is not setup on the App Service. 
-
-Check the environment variables MSI_ENDPOINT and MSI_SECRET exist using [Kudu debug console](https://azure.microsoft.com/en-us/resources/videos/super-secret-kudu-debug-console-for-azure-web-sites/). If these environment variables do not exist, MSI is not enabled on the App Service. Note that after enabling MSI, you need to restart your WebApp.
-
-### Common issues across environments:
-
-1. Access denied
-
-The principal used does not have access to the Key Vault. The principal used in show on the web page. Grant that user (in case of developer context) or application "Get secret" access to the Key Vault. 
+## Folders introduction
```

Comment:
Please turn this into a sentence like: You will find the following folders: key-vault-node-quickstart-3, which references the X.X SDK and keyvault-node-quickstart-v4, which uses the X.X version of the SDK.